### ISSUE TITLE
Schematron error when ESPD Response contains Lot Information

### ISSUE THREAD

**Issue by jerouris** [OP-TED/ESPD-EDM#112]
The schematron used for validation of ESPD Response 1.0.2 contains the following rule which is problematic:
```xml
<reports>
      <error xsi:type="BAR">
         <description>The current version of the ESPD Service does not allow for the specification of multiple lots. To ensure conformace with the current version of the ESPD Service the ID sub-element inside the element ProcurementProjectLot MUST be 0 (zero).</description>
         <location>xml:84:0</location>
         <test>( not(cbc:ID!='0') )</test>
      </error>
   </reports>
```
However, the ESPD Response has a field specifically for defining lots which uses the specific UBL Element.
The Schematron should be updated to reflect the correct business rule for ESPD Response 1.0.2
[Artifact and report as a zip file](https://github.com/ESPD/ESPD-EDM/files/1465910/artifact.and.report.zip)

**Comment by paulakeen** [comment]
This is being corrected in the Version 2.0.1 REGULATED version. We can correct this also for version 1.0.2 but in order to not having to create a new version 1.0.3 or similar I propose to create a folder with bug-fixes for version 1.0.2 ("Previous-versions-fixed-bugs") inside the distribution package of version 2.0.1. 

What do you think about my proposal?

**Comment by paulakeen** [comment]
After revewing this story and origins of this issue the conclusions are:

1. ESPD-EDM-V1.0.2 *cannot* be used for procurement procedures divided in lots.

2. ESPD-EDM-V2.0.0 is intended to move from the GROWTH's proprietary specification towards the UBL-2.2 specification. It was decided to have two "flavours" of XML implementations, both based on the same common set of UBL schemas:

- ESPD-EDM-2.0.0 REGULATED, with the objective of ensuring backwards compatibility with the previous ESPD-EDM-1.0.2. This ESPD-EDM-2.0.0 REGULATED cannot be used for procedures divided into Lots, otherwise the compatibility would be broken.

- ESPD-EDM-2.0.0 Self-contained, which is much more flexible and can be used for procedures divided into Lots, but is not compatible with the previous version 1.0.2. 

**In conclusion: the Schematron rule is correct and should not be modified.**

**Comment by paulakeen** [comment]
Angsar, Jerry and Ulf say: "
Although the ESPD-EDM model was designed to support multiple lots, the current version of the ESPDService, however, still does not fully uses this feature and permits only to describe them. The rule for the ESPD response is that “The value `0 MUST be specified to indicate that this procurement project has no lots.” That is at least the way it is written in the ESPD-EDM-V1.0.2 specs [1] in the BIS [2] and the Norwegian ESPD v1.0.2 specs [3]. Also the DG Grow ESPD-service and our eSENS ESPD service allow to put values into the field in order to describe the lots you are applying for in the response. Otherwise it would not make sense to have this element and we could delete it. In conclusion we think that the schematron rule and the corresponding validation service for version 1.0.2 is wrong as it is to strict and always require a “0” to be put. We would recommend to align the resolution on issues 112 and 127 accordingly."

**Comment by paulakeen** [comment]
Agree. Please confirm the following text, in section "III.10 Lots - Regulated ESPD" is then correct (and would replace the text currently posted in v2.0.1):

REQUIREMENT
At least one Lot must be always instantiated in the Regulated ESPD XML document.

If the procurement procedure is not divided into Lots the only lot present in the Regulated ESPD XML document represents the procedure, and its Identifier value must be '0'.

If the procedure is divided into Lots you can create multiple instances of the element ProcurementProjectLot and use it to identify and describe the Lots.

**Comment by lotzmann** [comment]
We confirm the proposed change for section "III.10 Lots - Regulated ESPD".
In order to maintain compatibility with ESPD 1.0.2, the respective schematron rule in the GITB validator needs to be updated, as it still only passes value "0" (i.e. the rule attached in the initial post is still active). Could you please trigger this amendment?

**Comment by paulakeen** [comment]
ESPD v.2.0.2 schematron files are accepting more 0 or more lots. The only constraint imposed is that when more than one lot are specified the identifiers of each lot are unique. 

However we've just realised that the documentation was not updated as agreed. This will be corrected in v.2.0.3.

**Comment by mireiaBM** [comment]
This issue was solved in version 2.1.0 of the ESPD-EDM
